### PR TITLE
Youtube link

### PR DESCRIPTION
**Pull request by Erik-Handeland**
# Description

Add a summary of the change and which issue is fixed, including motivation, context, dependencies, etc. 

# Checklist:

- [ ] Code follows the style guidelines
- [ ] Code commented in relevant places
- [ ] Changes to necessary documentation made
- [ ] Has test cases for code.

# Tests Performed:

- [ ] Upload data to Datastore
- [ ] Retreive data from Datastore
- [ ] Map Visualization 
- [ ] Network Map Visualization 
- [ ] List other manual test cases performed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,85 @@
...
 
-1. Download `file1.pcap` and `file2.pcap` from GDrive's PCAP Files folder.
-2. Put in repo under `resources/files/file1.pcap` and `resources/files/file2.pcap`, respectively. 
+1. Place any files inside of the  `resources/files` folder
+2. Update `pcap-uploader.html` to include the path to any added PCAP files
 
-## To add IP information for PCAP parser testing:
+### To add IP information for PCAP parser testing:
 
 1. Download `file2.txt` from GDrive's PCAP Files folder.
```
Comment (Erik-Handeland) [inline README.md]: Should we upload the test files? otherwise the project will fail to load

File: README.md [hunk 2]
```diff
@@ -1,38 +1,85 @@
...
 
-## To use geolocation: 
+### To use geolocation: 
 
 1. Download `GeoLite2-City.mmdb` from GDrive.
 2. Put in repo under `resources/GeoLite2-City.mmdb`.
 
-## MaxmindDB perf: 
+### Keystore
+API keys are handled by Keystore, You will need to modify `KeystoreDao` in order to run the project
```
Comment (Erik-Handeland) [inline README.md]: @thorncorona Not sure how correct this is. Will users just need to add something similar to the keystone.json file into the resource file? If so could with upload add a blank one with XXX for the API key so it's easy for a user to replace and get the project functioning

File: README.md [hunk 3]
```diff
@@ -1,38 +1,85 @@
...
+### Team
 Authors: handeland@google.com, jevingu@google.com, mnuzen@google.com 
 Reviewers: arunkaly@, promanov@
 
+### Built With
+- [Maven](https://maven.apache.org)
+- [Java](https://java.com/en/download/faq/java8.xml)
+- [App Engine](https://cloud.google.com/appengine/docs/standard/java)
+
 ## Setup:
```
Comment (Erik-Handeland) [inline README.md:52]: @mnuzen @thorncorona This section could use some work to make it more fluid. I remove any mention of the cloud bucket as that will no longer be available

**Review by mnuzen — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mnuzen) [reply README.md]: I think they've been uploaded right? Which ones are missing?

Comment (Erik-Handeland) [reply README.md]: They were uploaded to the cloud bucket, but not GitHub.

**Pull request merged**